xtask/config.toml.sample

```diff
@@ -4,3 +4,61 @@ user = ""
 # The personal access token to use for authentication. Needs 'repo' scope.
 # See: https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token
 token = ""
+
+[ci]
```

Comment:
This should not be part of `config.toml`, which is for the user to customize. We could consider having this data in a `toml` file, but this one needs to be copied (from `config.toml.sample` to `config.toml`) and I don't see a reason to do that for `xtask ci`.

Reply:
Sorry, I am confused by this. So do we need to put it in a separate `toml` file, or just remove the sample config, or put in another sample config (something like `tests.toml.sample`) or something else ?

Reply:
I think it would be good to simply have four separate commands:

* `cargo xtask ci stable`
* `cargo xtask ci msrv`
* `cargo xtask ci nightly`
* `cargo xtask ci`

Where the first three do pretty much exactly what's in the `.builds` scripts, and `cargo xtask ci` without a subcommand runs all of them. None of this needs to be configurable.

Reply:
okay. I will create a new PR with this only. I will just hardcode the build scripts into the code then.